pkg/services/sqlstore/migrator/dialect.go

```diff
@@ -53,6 +53,17 @@ type Dialect interface {
 	UpsertSQL(tableName string, keyCols, updateCols []string) string
 	UpsertMultipleSQL(tableName string, keyCols, updateCols []string, count int) (string, error)
 
+	// Concat returns the sql statement for concating multiple strings
+	// Implementations are not expected to quote the arguments
+	// therefore any callers should take care to quote arguments as necessary
+	Concat(string, ...string) string
```

Comment:
The implementation might get simpler if the signature becomes `Concat(...string) string` instead